Nodejs/Tests/Core.UI/SnippetsTests.cs

```diff
@@ -215,14 +214,12 @@ class Declaration {
             foreach (var decl in snippet.Declarations) {
                 Console.WriteLine("Declaration: {0}", decl.Replacement);
                 Keyboard.Type(decl.Replacement);
-                Keyboard.Type("\t");
+                Keyboard.Type("→");

```

Comment:
Edit.CompleteWord does not work here. Edit.CompleteWord only completes the word if there is only one match in the completion set.